app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreen.kt

```diff
@@ -85,51 +90,81 @@ fun SplashScreen(viewModel: SplashViewModel = hiltViewModel()) {
                     )
                 }
         )
-        AnimatedVisibility(visible = showLoginComponents, enter = fadeIn()) {
-            LoginComponents()
-        }
+        AnimatedVisibility(visible = showLoginComponents, enter = fadeIn()) { LoginComponents() }
+        Loading(showLoading = shouldShowLoading)
+        ErrorAlertDialog(
+            errorState = shouldShowError,
+            title = stringResource(id = R.string.oops),
+            buttonText = stringResource(id = android.R.string.ok),
+            onClickButton = { viewModel.resetError() }
+        )
```

Comment:
And then update this accordingly 😄 
```suggestion
        error?.let { error ->
            ErrorAlertDialog(
                errorState = error,
                title = stringResource(id = R.string.oops),
                buttonText = stringResource(id = android.R.string.ok),
                onClickButton = { viewModel.resetError() }
            )
        }
```